utils/wasm-builder/src/cargo_command.rs

```diff
@@ -67,6 +69,11 @@ impl CargoCommand {
         self.profile = profile;
     }
 
+    /// Set features.
+    pub fn set_features(&mut self, features: &[String]) {
```

Comment:
`&[&str]` is more general, but not critical.
```suggestion
    pub fn set_features(&mut self, features: &[&str]) {
```